projects/angular/src/data/datagrid/datagrid-filter.spec.ts

```diff
@@ -140,7 +140,9 @@ export default function (): void {
 
       it('has a button with the correct aria-label', function () {
         const toggle: HTMLButtonElement = context.clarityElement.querySelector('.datagrid-filter-toggle');
-        expect(toggle.getAttribute('aria-label')).toBe('Toggle user filter');
+        context.fixture.whenStable().then(() => {
```

Comment:
Use async/await.

Reply:
async/await is not working

Reply:
This test is only passing because the expectation is not being evaluated. You can put any expected value and the test will pass.

Reply:
updated the spec